src/CommonGLPI.php

```diff
@@ -1031,7 +1049,7 @@ class='btn btn-sm btn-icon btn-ghost-secondary'
 
             echo "</div>";
 
-            if ($this instanceof CommonITILObject) {
+            if (static::$showTitleInNavigationHeader) {
                 echo "<h3 class='navigationheader-title strong d-flex align-items-center'>";
                 echo "<span class='me-1'>" . $this->getStatusIcon($this->fields['status']) . '</span>';
```

Comment:
This change could permit to use this feature on all items, not only those that are implementing `getStatusIcon()`
```suggestion
                if (method_exists($this, getStatusIcon) && $this->isField('status')) {
                    echo "<span class='me-1'>" . $this->getStatusIcon($this->fields['status']) . '</span>';
                }
```